Libraries Tasmania.js

```diff
@@ -0,0 +1,781 @@
+{
+	"translatorID": "44699e59-a196-4716-ae33-141ec605e394",
+	"label": "Libraries Tasmania",
+	"creator": "Tim Sherratt (tim@timsherratt.org)",
+	"target": "^https?://librariestas\\.ent\\.sirsidynix\\.net\\.au/client/en_AU/",
+	"minVersion": "5.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2022-06-18 04:45:03"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2022 Tim Sherratt
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+const formatMapping = {
+	photograph: "artwork",
+	"image (online)": "artwork",
+	eMusic: "audioRecording",
+	cd: "audioRecording",
+	eAudiobook: "audioRecording",
+	dvd: "videoRecording",
+	"archived website": "webpage"
+};
+
+function detectWeb(doc, url) {
+	let catType = url.match(/en_AU\/(library|tas|names)\/search/);
+	if (url.includes("results") && getSearchResults(doc, true)) {
+		return "multiple";
+	}
+	else if (catType == "tas" || url.includes("ARCHIVES_")) {
+		return "manuscript";
+	}
+	else if (catType == "names" || url.includes("NAME_INDEXES")) {
+		return "manuscript";
+	}
+	else {
+		var formats = doc.querySelectorAll("div.displayElementText.text-p.LOCAL_FORMAT");
```

Comment:
Are all these classes necessary or would just `".LOCAL_FORMAT"` work? For stability if the site changes.

Reply:
The `.LOCAL_FORMAT` class is also used in the field label, but simplifying to `.displayElementText.LOCAL_FORMAT` seems ok.